versions/3.0.md

```diff
@@ -191,6 +192,30 @@ Contact information for the exposed API.
 
 Field Name | Type | Description
 ---|:---:|---
+<a name="languagesCurrent"></a>current | `string` | The language code for the current document.
+<a name="supportedLanguages"></a>supportedLanguages | [Parameter Object] | A parameter object of type `in: header` or `in: query` used to describe how to fetch the OAS document in the language allowed by the `enum`
+
+Example:
+```yaml
+languages:
+  current: en-us
+  supportedLanguages:
+    in: query
+    name: lang
+    type: string
+    enum:
+      - en-us
+      - es
```

Comment:
Hmm, why not simply a map of links?

```yaml
languages:
  current: en-us
  supportedLanguages:
    es: ./openapi.yaml?lang=es
    en-us: ./openapi.yaml?lang=en-us
```
Is this trying to avoid mentioning the own file-name?